config/lib/helpers/cache.js

```diff
@@ -5,4 +5,12 @@ module.exports = {
     name: 'campaigns',
     ttl: parseInt(process.env.GAMBIT_CAMPAIGNS_CACHE_TTL, 10) || 3600,
   },
+  defaultTopicTriggers: {
+    name: 'defaultTopicTriggers',
+    ttl: parseInt(process.env.GAMBIT_DEFAULT_TOPIC_TRIGGERS_CACHE_TTL, 10) || 3600,
```

Comment:
I didn't see this ENV variables in the `.env.example` file. They could be included in the documentation PR too.